packages/amplify-cli/src/lib/input-params-manager.js

```diff
@@ -26,7 +26,12 @@ function normalizeKey(key) {
 }
 
 function normalizeValue(key, value) {
-  const normalizedValue = JSON.parse(value);
+  let normalizedValue = value;
+  try {
+    normalizedValue = JSON.parse(value);
+  } catch (e) {
+    // do nothing
```

Comment:
This change seems unrelated to the PR. What's this for?

Reply:
It's in the description of the PR. 
It's too restrictive to only allow Json string as parameters. 
This update allows plain string as parameter, which is used for things such as sample's '--app' option.

Reply:
I'm not sure this is the best way to allow a plain string parameter...do we want to return the value if an invalid JSON is passed?

Reply:
If the value can not be parsed as JSON, then it's passed as plain string.